pyam/worldbank.py

```diff
@@ -63,7 +57,7 @@ def read_worldbank(model="World Bank", scenario="WDI", **kwargs) -> IamDataFrame
         unit="n/a",
         region="country",
     )
-    # TODO use wb.get_indicators to retrieve corrent units (where available)
+    # TODO use wb.get_indicators to retrieve correct units (where available)
```

Comment:
Thanks for clarifying this, I wasn't sure if this should be `current` or `correct` :)